src/components/WelcomeForm/Message.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import styles from "./Message.module.css";
+
+const Message = ({ user }) => {
+  if (!user.userName && !user.age) {
+    return null;
+  }
+
+  if (isNaN(user.age) || user.age == "") {
+    return (
+      <div>
+        <p className={styles.error}>The age value is incorrect, please try again!</p>
+      </div>
+    );
+  } else if (Number.parseInt(user.age) <= 18) {
```

Comment:
Here since you already has a numeric value the parseInt is not needed, also the condition will show the incorrect message if the user enters 18, we should allow to enter if the user is >= 18